### DISCUSSION TITLE
How to read data without DELIMITER?

### DISCUSSION THREAD

**Discussion by sharkrice** [kenjdavidson/react-native-bluetooth-classic#158] [Q&A] [answered]
How to read data without delimiter? 
I have a device and its data does not have any separators.
If I set the delimiter in the options of connect to an empty string, I will receive countless empty strings.
How can there be no delimiter when receiving data. ?

**Comment by kenjdavidson** [comment, marked as answer]
What version are you using?

I thought I fixed that issue. Looking at the code I am making sure that the buffer has content before reading.  So it should break out if the buffers empty.

This isn't the best, like I told the last guy, it's called delimited string connection for a reason. But it should work, he had confirmed it. #154

You can always write your own connection if you need finer control.

**Reply by sharkrice** [reply]
Solved after updating to the latest version

**Comment by kenjdavidson** [comment]
OK cool. Thanks for the update.
